docs/getting-started/products/ybribe/how-it-works.md

```diff
@@ -0,0 +1,78 @@
+# How it Works
+ 
+
+## Scenario #1: Voters Looking to Claim Bribes
+
+veCRV holders can sell their gauge votes to the highest bidder each week by 1- voting on the bribed gauge with the highest APR and then choosing to 2- claim any applicable rewards
```

Comment:
veCRV holders can sell their gauge votes to the highest bidder each week by `1 -` voting on the bribed gauge with the highest APR and `2 - claiming any applicable rewards.`

forgot the period and I'd add spacing to the -.